### PR TITLE
Allow custom logstash plugins.

### PR DESCRIPTION
**Pull request by jmcarp**
This uses a format that's slightly different from the one used for elastic and kibana plugins, but it's useful to be able to specify arbitrary arguments to the installer like `--version`. WDYT?

### CODE REVIEW

**Comment by axelaris** [comment]
Hi @jmcarp and happy New Year!
What's the status of signing CLA from your side? Did our lawyers had a chance to talk?

**Comment by mogul** [comment]
Hi @axelaris, and happy new year to you too! 

[oops, wrong info]

**Comment by mogul** [comment]
Sorry, I had your CLA confused with Pivotal's. I'll check!

**Comment by jmcarp** [comment]
Merge conflicts resolved.

**Comment by jmcarp** [comment]
@Infra-Red: can I trouble you to take a look at this when you have time? Looking forward to building this release from upstream and not from our fork 😄

File: jobs/ingestor_syslog/spec [hunk 1]
```diff
@@ -47,10 +52,6 @@ properties:
   logstash_ingestor.syslog_tls.ssl_key:
     description: Syslog-TLS SSL key (file contents, not a path) - required if logstash_ingestor.syslog_tls.port set
 
-  logstash_ingestor.relp.port:
-    description: Port to listen for RELP messages
-    default: 2514
-
```
Comment (Infra-Red) [inline jobs/ingestor_syslog/spec:53]: @jmcarp Overall it looks good, but could you add this part back, I think that they are lost during the sync with upstream.

**Review by Infra-Red — CHANGES_REQUESTED** [review]

**Comment by jmcarp** [comment]
Thanks for catching that mistake, just reverted it.

**Comment by Infra-Red** [comment]
LGTM after fixing. Thanks @jmcarp

**Pull request merged**